### ISSUE TITLE
prte fails to start due to missing "curl" symbols if PMIx configure --with-curl

### ISSUE THREAD

**Issue by rhc54** [openpmix/openpmix#2142]
The `src/mca/common/sse` component depends upon `libcurl` and only builds if `--with-curl` is given and a suitable `libcurl` found at the indicated (or standard) location. Once it is built, however, `prte` fails to run with an error:
```text
$ prte
dyld: Symbol not found: _curl_global_cleanup
  Referenced from: /Users/rhc/pmix/build/clean/lib/libpmix.0.dylib
  Expected in: flat namespace
 in /Users/rhc/pmix/build/clean/lib/libpmix.0.dylib
Abort trap: 6 (core dumped)
$
```

**Comment by rhc54** [comment]
This still persists on both master and v4.0.1

**Comment by rhc54** [comment]
@bwbarrett You can replicate this as stated above - just build PMIx `--with-curl` and have `libcurl` available. I did try a fix for it in https://github.com/openpmix/openpmix/pull/2171 but without success. Appreciate the assist!

**Comment by bwbarrett** [comment]
Closing as the issue is fixed in master and common/sse is not in the v4.0 or v4.1 release branches.
